### PR TITLE
User fix

### PR DESCRIPTION
**Pull request by lzychowski**
- recovered removed code for tutorial
- added error handling for running zerocrm without logging in

### logged in
screen shot 2018-06-05 at 4 45 07 pm

### not logged in (for local dev)
screen shot 2018-06-05 at 4 45 03 pm

### CODE REVIEW

File: views/settings.ejs [hunk 1]
```diff
@@ -49,7 +49,9 @@
         </ol>
         <div class="user">
           <div class="user-image">
-              <img src="<%- user._json.picture %>">
+              <%if (user) { %>
```
Comment (jcenturion) [inline views/settings.ejs]: How it looks like when the user is not logged-in?

**Comment by jcenturion** [comment]
`added error handling for running zerocrm without logging in` => is this for running ZeroCRM locally?

**Comment by lzychowski** [comment]
@jcenturion yes, the error handling is for running locally without the website running and without the user object being populated

**Review by jcenturion — APPROVED** [review]

**Comment by jcenturion** [comment]
LGTM

**Pull request merged**